package.json

```diff
@@ -3,10 +3,11 @@
   "repository": "github:vincent-peugnet/wcms",
   "license": "MIT",
   "dependencies": {
+    "@sentry/browser": "^5.7.1",
     "codemirror": "^5.49.0"
   },
   "devDependencies": {
-    "@release-it/bumper": "^1.0.5",
+    "@sentry/cli": "^1.49.0",
```

Comment:
The CLI isused to create releases on Sentry. This is how we can upload the sourcemaps to Sentry